playbooks/common/openshift-master/config.yml

```diff
@@ -244,6 +244,8 @@
   - fail:
       msg: "openshift_master_session_auth_secrets and openshift_master_encryption_secrets must be equal length"
     when: (openshift_master_session_auth_secrets is defined and openshift_master_session_encryption_secrets is defined) and (openshift_master_session_auth_secrets | length != openshift_master_session_encryption_secrets | length)
+  - name: Install OpenSSL package
+    action: "{{ansible_pkg_mgr}} pkg=openssl state=present"
```

Comment:
Before we are ready to merge, please remove the commit that adds this, since there is already a PR for it here: https://github.com/openshift/openshift-ansible/pull/982